README.rst

```diff
@@ -105,6 +103,17 @@ news and updates.
 .. _its Web site: https://beets.io/
 .. _@b33ts: https://twitter.com/b33ts/
 
+Contact
+-------
+* Encountered a bug you'd like to report or have an idea for a new feature? Check our our `issue tracker`_! If your issue or feature hasn't already been reported, please `open a new ticket`_ and we'll be in touch with you shortly.
+* Need help/support, would like to start a discussion, or would just like to introduce yourself to the team? Check out our `forums`_!
```

Comment:
Are you wanting support questions on github or discourse?

Reply:
Here's how I'd break it down: if you have a concrete problem to report (e.g., a traceback), make an issue. If you are not sure whether it's your fault or beets's, start a thread on Discourse and we might redirect you to an issue if it looks like a bug. So yeah, I think that means what you've said here: "help/support" goes to the forum.